src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -212,7 +212,7 @@ export default function TokenDetails({
             </BreadcrumbNavLink>
             <TokenInfoContainer data-testid="token-info-container">
               <TokenNameCell>
-                <CurrencyLogo currency={detailedToken} size="32px" hideL2Icon={false} />
+                <PortfolioLogo currencies={[detailedToken]} chainId={detailedToken.chainId} size="32px" />
```

Comment:
and here

Reply:
typechecking gets ugly trying to memo here so I'm gonna fix this one in the ticket i mentioned above